jihyeon/middleware/auth.js

```diff
@@ -0,0 +1,16 @@
+const jwt = require("jsonwebtoken");
+
+const validateToken = async (req, res, next) => {
+  try {
+    const token = req.headers.authorization;
+    if (!token) return res.status(403).json("ACCESS_DENIED");
+
+    const decoded = jwt.verify(token, process.env.SECRETKEY);
+    req.user = decoded.user_id;
+    next();
+  } catch (err) {
+    next(err);
+  }
+};
```

Comment:
여기서 next(err)를 하면 어떤 일이 벌어지나요?? 
지금 상황에서는 정상적으로 작동하지 않을 것 같습니다. 
next 대신에 return으로 적절한 res을 반환해 주시면 좋을것 같습니다.
그리고 왜 next(err)을 하면 안 될지도 고민해 보세요.
```suggestion
  } catch (err) {
    return res.status(적절한 코드).json(적절한 메시지);
  }
};
```